.drone.yml

```diff
@@ -51,9 +32,51 @@ trigger:
     exclude:
     - promote
 
+---
+kind: pipeline
+name: publish-linux
+type: docker
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: publish-linux
+  image: rancher/hardened-build-base:v1.16.4b7
+  commands: 
+  - docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+  - DRONE_TAG=${DRONE_TAG} make publish-linux-amd64
+  environment:
+    DOCKER_PASSWORD:
+      from_secret: docker_password
+    DOCKER_USERNAME:
+      from_secret: docker_username
+  volumes:
+    - name: docker
+      path: /var/run/docker.sock
+
+  when:
+    event:
+      - tag
+    instance:
+      - drone-publish.rancher.io
+    ref:
+      - refs/head/master
+      - refs/tags/*
```

Comment:
Let's delete this pipeline and move this back into the one above

Reply:
cleaned up this pipeline and folded it back into linux-amd64